testing/tests/consensus.rs

```diff
@@ -299,7 +307,11 @@ where
 /// Submits proposals for 0..NUM_VIEWS rounds where `node_id` is the leader
 fn test_validating_proposal_queueing_round_setup<
     TYPES: NodeType<ConsensusType = ValidatingConsensus>,
-    ELECTION: Election<TYPES, LeafType = ValidatingLeaf<TYPES>>,
+    ELECTION: Election<
```

Comment:
This is a mouthful. Wish we could type alias the trait within an `impl` block.

Reply:
Me too! Maybe after the big refactor, we could try to make them look better. I think it's okay for now since we are going to change the `Election` trait anyway.